class UserNotFound(Exception):
    pass


class UserAlreadyExists(Exception):
    pass
